packages/react-server-examples/bike-share/package.json

```diff
@@ -7,35 +7,21 @@
   "author": "Doug Wade <doug.wade@redfin.com>",
   "scripts": {
     "clean": "rm -rf build __clientTemp",
-    "start": "npm run clean && npm run styles && react-server start",
-    "styles": "node-sass styles/index.scss build/styles/index.css && node-sass styles/network.scss build/styles/network.css",
-    "test": "xo && nsp check"
+    "start": "npm run clean && react-server start --hot",
```

Comment:
It seems like a lot of overhead to add a whole new file to eliminate one flag...